metron-analytics/metron-profiler-client/README.md

```diff
@@ -41,13 +41,20 @@ Retrieve all values of 'snort-alerts' from '10.0.0.1' that occurred on 'weekdays
 PROFILE_GET('snort-alerts', '10.0.0.1', 1, 'MONTHS', 'weekdays')
 ```
 
-### Period Duration
+### Configuration
 
-*WARNING*
+By default, the Profiler creates Profiles with a period duration of 15 minutes. This means that data is accumulated, summarized and flushed every 15 minutes. The Client API must also have knowledge of this duration to correctly retrieve the profile data. If the client API is expecting 15 minute periods, it will not be able to read data generated by a Profiler that has been configured with a 1 hour period.
 
-By default, the Profiler creates Profiles with a period duration of 15 minutes.  This means that data is accumulated, summarized and flushed every 15 minutes.  The Client API must also have knowledge of this duration to correctly retrieve the profile data.  If the client API is expected 15 minute periods, it will not be able to read data generated by a Profiler that has been configured with a 1 hour period.
+The period duration can be configured in the Profiler by altering the Profiler topology's static properties file. The Stellar Client API can be configured by setting the following properties in Metron's global configuration.
 
-The period duration can be configured in the Profiler by altering the Profiler topology's static properties file.  The Stellar Client API currently provides no means to configure the period duration and defaults also to 15 minutes.  This means that the Stellar Client API can only read profiles with a period duration of 15 minutes.  This is a known limitation that has not yet been addressed.
+| Key                                   | Description                                                                                                                        | Required | Default  |
+| ------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------- | -------- | -------- |
+| profiler.client.hbase.table           | The name of the HBase table used to store profile data.                                                                            | Optional | profiler |
+| profiler.client.column.family         | The name of the HBase column family used to store profile data.                                                                    | Optional | P        |
+| profiler.client.hbase.table.provider  | The name of the HBaseTableProvider implementation class.                                                                           | Optional |          |
+| profiler.client.period.duration       | The duration of each profile period.  This value should be defined along with `profiler.client.period.duration.units`.             | Optional | 15       |
+| profiler.client.period.duration.units | The units used to specify the profile period duration.  This value should be defined along with `profiler.client.period.duration`. | Optional | MINUTES  |
```

Comment:
In actuality, the Profiler does match the same scheme as the client (using duration and units).  The docs for the Profiler were not updated.  I will fix that now.  Good catch.